vnc.html

```diff
@@ -274,6 +274,25 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br>VNC</h1>
         </div>
     </div>
 
+    <!-- Server Key Verification Dialog -->
+    <div class="noVNC_center noVNC_key_verification_layer">
+    <div id="noVNC_verify_key_dlg" class="noVNC_panel"><form>
+        <ul>
```

Comment:
Using a list here is a bit odd. But I can see that this is already done for the credentials dialog, so I can do that cleanup later.